projects/spectator/tsconfig.lib.json

```diff
@@ -12,7 +12,7 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "types": [
-      "@types/jasmine"
```

Comment:
@dirkluijk in the past it works for me only because I had both jest and jasmine installed. We should test it on a clean project.

Reply:
I cannot load both `@types/jasmine` and `@types/jest` at the same time. That is because `@types/jest` redefines Jasmine namespace and typings. See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/v16/index.d.ts#L178.